src/FileGetContents.php

```diff
@@ -264,24 +334,29 @@ protected function getMergedStreamContext($url)
     }
 
     /**
-     * @deprecated
+     * @deprecated since 1.1.0 and will be removed in 2.0.0.
      */
     public static function getSystemCaRootBundlePath()
     {
+        @trigger_error(
+            'Deprecated since 1.1.0. Use `Composer\CaBundle\CaBundle::getSystemCaRootBundlePath()` instead.',
+            E_USER_DEPRECATED
+        );
+
         return CaBundle::getSystemCaRootBundlePath();
     }
 
     /**
-     * @deprecated
+     * @deprecated since 1.1.0 and will be removed in 2.0.0.
      */
     protected static function validateCaFile($contents)
```

Comment:
I would like a more helpful deprecation message here like the one just above, but I can't find any. There's actually no use of that method in our codebase so should this be exposed in a function or this shouldn't exist in the first place?